src/components/v5/common/ActionSidebar/partials/AmountField/AmountField.tsx

```diff
@@ -99,93 +99,102 @@ const AmountField: FC<AmountFieldProps> = ({ name }) => {
             onChange={handleCleaveChange}
             style={{ width: `${inputWidth || 0.65}rem` }}
           />
-          <div className="sm:relative w-full">
-            <button
-              type="button"
-              ref={relativeElementRef}
-              className={clsx(
-                'flex items-center gap-2 transition-colors md:hover:text-blue-400',
-                {
-                  'text-gray-900': selectedToken?.symbol,
-                  'text-gray-500': !selectedToken?.symbol,
-                },
-              )}
-              onClick={toggleTokenSelect}
-              aria-label={formatMessage({ id: 'ariaLabel.selectToken' })}
-            >
+          {tokenAddress ? (
+            <div className="flex items-center gap-2">
               <TokenIcon token={selectedToken || colonyTokens[0]} size="xs" />
               <span className="text-md">
                 {selectedToken?.symbol || colonyTokens[0].symbol}
               </span>
```

Comment:
Seems like this has been reapeated 3 times across this component, can we create a variable for this content?